_component_design/tables-basic.md

```diff
@@ -0,0 +1,156 @@
+- Text justification within each cell is content specific.
+  - Text should be left justified.
+  - Center justify columns that contain null/undefined (–), not applicable (n/a), or icons.
+  - Numbers should be right justified.
+- Only use units in row if they differ row by row and cannot be standardized. Units in body rows appear without parentheses.
+- Truncated text should appear in a [tooltip]({{site.baseurl}}/components/tooltip.html), enabling display on hover or click.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Default table specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/basic-tables/tables-default.svg)
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+## Condensed table
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+The use of a condensed table provides for views that display either many or few results. 
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Condensed table specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/basic-tables/tables-condensed.svg)
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+### Two-row / small tables
+
+Use the condensed table style for small sets of data which are best represented in tabular form. In cases such as this, tables are preferred over bullet lists, or key-value lists.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Example: IP addresses currently applied in a cloud server"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/basic-tables/tables-small-condensed.svg)
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+## Table states 
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+### Loading
+
+As tables load, use a gray placeholder to represent the loading text. This pattern informs the user that the data is present but not yet available.
+
+{% endcolumn %}
+
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Example of table in loading state"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/basic-tables/tables-loading.svg)
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+### Empty
+
+Empty tables occur when no data is available or when a user has applied a filter that does not fit the table parameters. 
+
+At times no data is available, the empty state is presented with textual hints that direct the user towards possible solutions in the event an error has occurred.
+
+Examples of this direction include:
+- "… contact your account owner."
```

Comment:
pretty sure there needs to be whitespace above this line

Reply:
updated.